web/pdf_page_view.js

```diff
@@ -805,41 +781,37 @@ class PDFPageView {
   }
 
   cssTransform({
-    target,
     redrawAnnotationLayer = false,
     redrawAnnotationEditorLayer = false,
     redrawXfaLayer = false,
     redrawTextLayer = false,
     hideTextLayer = false,
   }) {
-    // Scale target (canvas), its wrapper and page container.
-    if (
-      (typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING")) &&
-      !(target instanceof HTMLCanvasElement)
-    ) {
-      throw new Error("Expected `target` to be a canvas.");
+    const { canvas } = this;
+    if (!canvas) {
+      return;
     }
-    if (!target.hasAttribute("zooming")) {
-      target.setAttribute("zooming", true);
-      const { style } = target;
+    if (!canvas.hasAttribute("zooming")) {
```

Comment:
Not directly related to this patch, but I wonder if we still need the following CSS rule?
https://github.com/mozilla/pdf.js/blob/a4eb8407c1b3695be0d1f91791f393655b476dd9/web/pdf_viewer.css#L95-L98
Note that since a little while back, all `canvas`es have those properties set:
https://github.com/mozilla/pdf.js/blob/a4eb8407c1b3695be0d1f91791f393655b476dd9/web/pdf_viewer.css#L88-L89